extensions/btl2cap/src/main/java/jolie/net/BTL2CapListener.java

```diff
@@ -78,5 +78,7 @@ public void shutdown() {
 			connectionNotifier.close();
 		} catch( IOException e ) {
 		}
+		super.clearInterpreter();
```

Comment:
We do these two lines a bit repetitively.. Maybe they could be put in an abstract class that the relevant listeners inherit from, or maybe we could even use WeakReference for the interpreter references in the superclass and the location value?